python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -918,6 +918,18 @@ def test_composite_channel_multi_output(self, ray_start_regular_shared):
         compiled_dag.teardown()
 
 
+def test_driver_and_actor_as_readers(ray_start_regular):
+    a = Actor.remote(0)
+    b = Actor.remote(10)
+    with InputNode() as inp:
+        x = a.inc.bind(inp)
+        y = b.inc.bind(x)
+        dag = MultiOutputNode([x, y])
+
+    with pytest.raises(ValueError):
```

Comment:
Can you add the `match=` str here? Helps with readability in the test too, so we know why this case should fail.

Reply:
Done